distributed/scheduler.py

```diff
@@ -7399,14 +7399,11 @@ async def get_worker_logs(self, comm=None, n=None, workers=None, nanny=False):
         return results
 
     def log_event(self, name, msg):
-        event = (time(), msg)
-        if isinstance(name, list):
-            for n in name:
-                self.events[n].append(event)
-                self.event_counts[n] += 1
-        else:
-            self.events[name].append(event)
-            self.event_counts[name] += 1
+        if not isinstance(name, list):
+            name = [name]
+        for n in name:
+            self.events[n]  # init defaultdict
+            self.extensions["pubsub"].handle_message(name=n, msg=msg)
```

Comment:
This is currently ugly but the handle_message of the extension will append the event to the deque and inc the counter now